python/lsst/ts/standardscripts/maintel/laser_tracker/measure.py

```diff
@@ -0,0 +1,172 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["Measure", "AlignComponent"]
+
+import asyncio
+import enum
+
+import yaml
+from lsst.ts import salobj
+from lsst.ts.idl.enums.LaserTracker import LaserStatus
+from lsst.ts.observatory.control import RemoteGroup
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+from lsst.ts.observatory.control.remote_group import Usages
+
+
+# TODO (DM-38880) - Subsitute by class in idl.enum when available
+class AlignComponent(enum.IntEnum):
```

Comment:
Why not use the same enum defined in the align script? Also, what about the TMA?